ui/packages/catalog-ui-search/src/main/webapp/react-component/container/input-wrappers/radio/radio.tsx

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
✏️ Consider using the already existing radio button component: https://github.com/codice/ddf/blob/0adc9b6cc16e30f1c82852c0f0bc87207d02ec20/ui/packages/catalog-ui-search/src/main/webapp/component/input/radio/input-radio.view.js